package.json

```diff
@@ -3,7 +3,7 @@
   "displayName": "BPMN.io Editor",
   "description": "Edit BPMN 2.0 files, powered by bpmn.io tools.",
   "license": "MIT",
-  "version": "0.17.0",
+  "version": "0.17.1",
```

Comment:
Due to missing change here, it was released as `v0.17.1` to the VSCode marketplace :(

Reply:
Oops, I did not notice this. I'll remember it for the next time. Do we want to re-release as 0.18?

Reply:
Let's do an `v0.18.1` since `v0.18.0` tag is already taken 😓